site/docs/foundations/data-visualization/chart-accessibility.mdx

```diff
@@ -0,0 +1,100 @@
+
+<Image
+  src="/img/foundations/dataviz-monochromatic.png"
+  srcDark="/img/foundations/dataviz-monochromatic-dark.png"
+  alt="Diagram of best practices using categorical palette versus monochromatic palette."
+/>
+
+You can enhance your chart’s accessibility by adopting additional visual strategies other than color such as:
+
+- Pattern overlays for different categories
+- Direct labelling where possible
+- Adding a border with at least a 3:1 contrast ratio to distinguish between segments
+
+You can show chart data in a table or list nearby, in an overlay, or on another screen as a last resort to support screen readers. If the essential information is in a separate screen, then it could disrupt the cohesive flow of information. Screen reader users would need to navigate back and forth between the two parts of the text alternative for the complex image, which could disrupt the flow of information.
+
+For more information about text alternatives, read the [JPMC UX Accessibility Resources](https://go/text-alternatives#TextAlternatives-Compleximages) (Internal users only).
+
+### Overlapping data series
+
+Some chart types including area, bubble and scatter have data series that overlap in certain use cases. Having too many data series in your chart can add complexity and make it harder for users to extract important information. In cases with multiple overlapping areas, always maintain at least a 3:1 color contrast between any 2 overlapping areas.
+
+<Image
+  src="/img/foundations/dataviz-overlapping-data.png"
+  srcDark="/img/foundations/dataviz-overlapping-data-dark.png"
+  alt="Diagram of bubble chart to show colors overlapping."
+/>
+
+## Line patterns
+
+Color shouldn’t be the only visual cue. Include other visual cues such as text, patterns, or symbols to help the user differentiate between different data sets. Patterns improve readability, especially for those with color vision requirements.
+
+<Image
+  src="/img/foundations/dataviz-line-patterns2.png"
+  srcDark="/img/foundations/dataviz-line-patterns2-dark.png"
+  alt="Diagram of line chart using line patterns."
+/>
+
+## Pattern fills
+
+Consider using line patterns for multiple line series or pattern fills for 2-dimensional shapes, such as bars and slices. Pattern fills are always black and applied to solid categorical colors for charts. They’re beneficial in scenarios in which there is only one visual distinction between data points.
+
+<Image
+  src="/img/foundations/dataviz-pattern-fills2.png"
+  srcDark="/img/foundations/dataviz-pattern-fills2-dark.png"
+  alt="Diagram of donut chart using pattern fills."
+/>
+
+## Marker symbols
+
+In addition to line patterns and pattern fills, you can add shape or icon markers to make data points more distinctive. Varied marker symbols help differentiate lines. Marker symbols have a white border in both light and dark mode for accessibility.
```

Comment:
Also reading this with the diagram makes me think marker symbols only apply to line charts but scatter plots are a good use-case too